interfaces/core.go

```diff
@@ -265,20 +265,57 @@ type StaticInfo struct {
 	AppArmorUnconfinedSlots bool
 }
 
+type PlugServicesSnippetSection string
+
+const (
+	PlugServicesSnippetUnitSection    PlugServicesSnippetSection = "unit"
```

Comment:
Maybe we should use the same case as systemd uses. Like `Unit`, `Service`. For future refactoring, if we get rid of `text/template` and just use an abstract representation of systemd units, it will be just easier to index sections by their effective names.